services/core/shared/core/compat/sdk_v5/blockchainIndex.js

```diff
@@ -694,16 +740,23 @@ const init = async () => {
 		// Download genesis block
 		await getBlockByHeight(await getGenesisHeight());
 
+		// Start the indexing process (accounts)
+		await indexGenesisAccounts();
+		await cacheLegacyAccountInfo();
+
 		// Index all the delegate accounts first
 		await indexAllDelegateAccounts();
 
+		// Start the previously scheduled account updates
+		await triggerAccountUpdates();
+
 		// Start the indexing process (blocks)
 		await indexMissingBlocks();
 		await updateNonFinalBlocks();
 
-		// Start the indexing process (accounts)
-		await cacheLegacyAccountInfo();
-		await indexGenesisAccounts();
+		if (KEY_BASED_ACCOUNT_UPDATE === true) {
+			setInterval(triggerAccountUpdates, 15 * 1000); // ms
+		}
```

Comment:
Same as above.

Reply:
That code prevents from updating too many accounts at once but it needs to work with the logic mentioned above. So far there is a partial solution to that (working lists, but problems with the public key update) so it need more updates to enable it in production.